eng/Directory.Build.Data.targets

```diff
@@ -19,7 +19,7 @@
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
       <DirectoryPartofPath>$(CurrentCsprojPath.Substring($(CurrentCsprojPath.LastIndexOf("\sdk\"))))</DirectoryPartofPath>
       <DirectoryPartofPath>$(DirectoryPartofPath.Remove($(DirectoryPartofPath.LastIndexOf("\src"))))</DirectoryPartofPath>
-      <PackageProjectUrl>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/README.md</PackageProjectUrl>
+      <PackageProjectUrl>$(RepositoryUrl)/blob/master$(DirectoryPartofPath)/README.md</PackageProjectUrl>
```

Comment:
Why are you moving to master? We should actually build this release tag for this instead of pointing to master.

Reply:
If we don't want to do the tag we should also be able to get the sha from the SourceLink tools which generate a property with the value.

Reply:
There was a discussion around this yesterday after the release that concluded with the decision to point to master. It was assumed that the latest commitSHA will not point to the readme if there was no change to the readme in that commit, that does not seem to be the case. However we settled on pointing to master as that's what the other repos are doing.